Main.java

```diff
@@ -9,7 +10,20 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+  Scanner input = new Scanner(System.in);
+
+  // ask the user to enter the first integer
+  System.out.println("Please enter in two integers, on seperate lines, to divide. ");
+  int x = input.nextInt();
+  int y = input.nextInt();
+
+  // divide the two integers to find the sum
+  int z = (x / y);
+  int v = (x % y);
+
```

Comment:
While the variables x and y could have better names, it isn't awful because they are just numbers. However, z and v are really the quotient and the remainder respectively. Remember, variable names should always describe what they are storing or what they are being used for. Otherwise, the code looks great.